### DISCUSSION TITLE
Failed to load module script with version 2022b robot window.

### DISCUSSION THREAD

**Discussion by Shadow149** [cyberbotics/webots#5379] [Q&A]
Since trying to upgrade my project to 2022b, adding the javascript module import code:
```js
import RobotWindow from 'https://cyberbotics.com/wwi/R2022b/RobotWindow.js';
```
I get the following error in the console:
`Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.`

Would someone be able to help me out with this?
Thanks

**Comment by BenjaminDeleze** [comment]
If I understand correctly you are converting a project with a custom robot window right? From R2022a?

**Reply by Shadow149** [reply]
Yes, that is correct.

**Reply by BenjaminDeleze** [reply]
I cannot reproduce this issue.
Would it be possible to share your project such that I can have a look?
